vuepress/docs/next/tutorials/devops/caching-and-clustering.md

```diff
@@ -156,12 +165,15 @@ entando-de-app-wildfly: >-
 kubectl apply -f namespace-resources.yaml
 ```
 
-10. Run the Helm template to generate the application configuration file `my-clustered-app.yaml`:
+<!-- 
+10. Run the Helm template to generate an application configuration:
+=======
+
 ```
 helm template quickstart ./ > my-clustered-app.yaml
 ```
 
-11. Add environment variables for your deployed Redis instance to `EntandoApp` in `my-clustered-app.yaml`. The variables to create are:
+11. Add environment variables to the `EntandoApp` in the outcome of the Helm template command generated above (`my-clustered-app.yaml`) for the Redis address and Redis password for your deployed Redis instance. The variables to create are: -->
```

Comment:
super confusing run-on sentence. is the original not correct? like "Add address and password environment variables for your deployed Redis instance to `EntandoApp` in `my-clustered-app.yaml`, which is the outcome of the Helm template generation."
the command wasn't generated, right? the YAML file was generated ??
the `EntandoApp` is misleading, like only one exists

Reply:
this section is commented out, to be deleted